app/src/main/java/at/aau/edu/wizards/ui/gameboard/claim/GuessViewHolder.kt

```diff
@@ -1,17 +1,34 @@
 package at.aau.edu.wizards.ui.gameboard.claim
 
+import android.content.Context
+import android.os.VibrationEffect
+import android.os.Vibrator
+import androidx.core.content.ContextCompat
 import androidx.recyclerview.widget.RecyclerView
 import at.aau.edu.wizards.databinding.ItemGuessBinding
 
 class GuessViewHolder(
     private val binding: ItemGuessBinding,
     private val onClick: (Int) -> Unit,
+    private val context: Context
```

Comment:
you don't need to pass context. You can get a context from `binding.root.context`